java/code/src/com/redhat/rhn/common/db/datasource/xml/System_queries.xml

```diff
@@ -2261,6 +2261,32 @@ SELECT channel_arch_id
   </query>
 </mode>
 
+<mode name="xmlrpc_empty_profiles" class="com.redhat.rhn.frontend.dto.BootstrapSystemOverview">
+    <query params="org_id, user_id">
+        SELECT
+        1 AS selectable,
+        0 AS locked,
+        rhnServer.id,
+        rhnServer.name,
+        TO_CHAR(checkin, 'YYYY-MM-DD HH24:MI:SS') AS last_checkin,
+        rhnCPU.nrcpu as cpu_count,
+        rhnCPU.mhz as cpu_mhz,
+        rhnRAM.ram as ram,
+        (SELECT COUNT(*) FROM rhnDevice WHERE rhnDevice.server_id = rhnServer.id AND class = 'HD') AS disk_count
+        FROM rhnServer
+        INNER JOIN suseMinionInfo on rhnServer.id = suseMinionInfo.server_id
+        LEFT JOIN rhnServerEntitlementView ON rhnServerEntitlementView.server_id = rhnServer.id
+        LEFT JOIN rhnUserServerPerms ON rhnServer.id = rhnUserServerPerms.server_id
+        LEFT JOIN rhnServerInfo ON rhnServer.id = rhnServerInfo.server_id
+        LEFT JOIN rhnCPU on rhnServer.id = rhnCPU.server_id
+        LEFT JOIN rhnRAM on rhnServer.id = rhnRAM.server_id
+        WHERE rhnServerEntitlementView.label = 'bootstrap_entitled'
```

Comment:
also rhnServerEntitlementView is joined via "left join" and therefor "optional". I think this is wrong.

Reply:
^